laalaa/apps/advisers/importer.py

```diff
@@ -14,16 +14,16 @@
 logging.basicConfig(filename='adviser_import.log', level=logging.WARNING)
 
 
-cache = {}
-
-
 def cached(fn):
-    cache[fn.__name__] = {}
+    cache = {}
 
     def wrapped(name):
-        if name not in cache[fn.__name__]:
-            cache[fn.__name__][name] = fn(name)
-        return cache[fn.__name__][name]
+        if name not in cache:
+            cache[name] = fn(name)
+        return cache[name]
+
+    wrapped.cache = cache
+    wrapped.clear_cache = lambda: cache.clear()
```

Comment:
I think you can remove this `lambda` and just do `wrapped.clear_cache = cache.clear`